openeit/app.py

```diff
@@ -8,7 +8,7 @@
 
```

Comment:
I disagree with adding the actual app to the package (but again, this is a personal preference). My argument is, that the package is a framework, that is independent of the presentation, it supplies infrastructure for image reconstruction, connecting to hardware devices and coordinating those (openeit.controller which is admittedly perhaps already to specific), but it does *not* depend on a GUI toolkit or specific UI.

In this sense dashboard.py is an application using the framework openeit, I plan to build dashboard_qt.py that reimplements the dashboard using Qt, I would not want to have that code hanging in the OpenEIT package – both dashboard front-ends could be supported by the same backend OpenEIT!